packages/react-jss/src/createUseStyles.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react'
-import isInBrowser from 'is-in-browser'
+import {isInBrowser} from 'jss'
```

Comment:
tbh I don't really want this to become a public API of jss

Reply:
I am fine with having it in both packages twice, unless its needed in more places, then it should be a separate package

Reply:
I agree with you I will add it in both of the packages `jss` and `react-jss`

Reply:
If we put it into a separate package, what name should be for this package? something like `jss-utils`?

Reply:
if we would do that, why not just use that package that we have now?

Reply:
Actually, I wanted to save a few bytes, but I think this hasn't more worth

Reply:
I will close this PR, are you agree?

Reply:
yep

Reply:
if you want to work on something, let me know and we will figure out a good issue
I recently added a discord if you want to chat https://discord.gg/e79XhqmaDH

Reply:
Great, my pleasure